hindsight/io_modules/derived_stream/redshift/psv.lua

```diff
@@ -13,13 +13,13 @@ local string    = require "string"
 
 setfenv(1, M) -- Remove external access to contain everything in the module
 
-local esc_chars = { ["|"] = "\\|", ["\r"] = "\\r", ["\n"] = "\\n" }
+local esc_chars = { ["|"] = "\\|", ["\r"] = "\\r", ["\n"] = "\\n", ["\0"] = "0" }
```

Comment:
@whd @mreid do you want it removed from the string or should we just discard the entire string as garbage